Header header
float64[] interface_signal
bool mode_switch
string interface_action